src/clr-angular/accordion/_variables.accordion.scss

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+$clr-accordion-text-color: $clr-color-neutral-1000 !default;
+$clr-accordion-active-background-color: $clr-global-selection-color !default;
+$clr-accordion-content-background-color: $clr-color-neutral-0 !default;
+$clr-accordion-header-background-color: $clr-color-neutral-50 !default;
+$clr-accordion-header-hover-background-color: $clr-color-neutral-200 !default;
+$clr-accordion-border-left-color: $clr-color-neutral-200 !default;
+$clr-accordion-responsive-breakpoint: map-get($clr-grid-breakpoints, sm) !default;
+$clr-accordion-error-color: $clr-color-danger-800 !default;
```

Comment:
Seems like global error and success colors could also find a home here?

Reply:
Hmm I didn't see a singular global error or success color I only see the numeric ranges, where would that be in the Sass?

Reply:
Updated with new global success and error color variables.